medicinski-razgledi.csl

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" delimiter-precedes-et-al="always" delimiter-precedes-last="always" initialize-with="" demote-non-dropping-particle="sort-only" default-locale="sl-SI">
```

Comment:
Are you sure you want to set the style to sl? All terms I'm seeing uses in references bloke et al and eds. are English

Reply:
Yes, it should be et al. Will change back to English.